src/migrations/versions/84796bdc2f77_add_jobstatus.py

```diff
@@ -0,0 +1,28 @@
+"""add_jobstatus
+Revision ID: 84796bdc2f77
+Revises: dbb81bd4d47f
+Create Date: 2024-10-01 08:09:42.808911
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "84796bdc2f77"
+down_revision = "dbb81bd4d47f"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute(
+        "CREATE TYPE jobstatus AS ENUM ('done', 'new', 'cancelled', 'removed', 'processing');"
```

Comment:
You don't need to know every possible type for this migration. Just change every value out of scope to `cancelled`and then convert to enum.

Reply:
(But good point that we need to handle this correctly)